package.json

```diff
@@ -291,6 +291,10 @@
             {
                 "command": "aks.aksRetinaCapture",
                 "title": "Run Retina Capture"
+            },
+            {
+                "command": "aks.aksKaito",
+                "title": "Deploy LLM with Kaito"
```

Comment:
I will rephrase in the merge, lemme quickly take care of this, LLM with Kaito is little off, will explain you in a call. :)

Reply:
I just copied from the UX mocks. anything is fine with me :)